ansible/playbooks/tools/reboot.yml

```diff
@@ -1,9 +1,11 @@
 ---
```

Comment:
Perhaps this playbook could be put in the `ansible/playbooks/` directory directly. Then we can use it nicely via the command line:
```
debops run site reboot -l host`
debops run common service/nginx reboot -l host
```